src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/Application.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
 
 @SpringBootApplication
+@EnableFeignClients
```

Comment:
do you need to define the packages too for s2s to work ?
```
@EnableFeignClients(basePackages =
    {
        "uk.gov.hmcts.reform.auth",
        "uk.gov.hmcts.reform.wacaseeventmanagement",
    })
```

Reply:
Are you asking it? I am confirming it as I write.

Reply:
We don't need it. Confirmmed.